nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+# Git as a Data Source
+
+The "Git™ as a Data Source" feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot; while leveraging the benefits that tools such as GitHub and GitLab already provide, including issue tracking, discussions, pipelines, and approvals.
+
```

Comment:
Would love an example this early on in the user guide.  

...tools such as GitHub and GitLab already provide, including issue tracking, discussions, pipelines, and approvals. For example, imagine being able to have users approve the YAML data that is used for Nautobot _config contexts_ along with running tests on that data, or having users approve Jinja2 templates that are used for Nautobot _export templates_ (just as a few examples).  That's what Git as a Data Source offers.